app/src/sharedTest/java/org/oppia/android/app/player/state/StateFragmentTest.kt

```diff
@@ -1468,6 +1472,19 @@ class StateFragmentTest {
     }
   }
 
+  @Test
+  @RunOn(TestPlatform.ESPRESSO) // TODO(#1612): Enable for Robolectric.
```

Comment:
This does not work on robolectric because we can't move past the drag and drop state of the exploration on robolectric yet. 

Also the logic for deleting progress is being tested in exploration activity test by the testing the altert dialog boxes.

Reply:
Ah, drag and drop is definitely an issue then. Thanks for clarifying.

And FWIW it should be possible to test AlertDialogs in Robolectric.